src/AppInstallerCLICore/Workflows/ImportExportFlow.cpp

```diff
@@ -139,16 +178,14 @@ namespace AppInstaller::CLI::Workflow
                 AICLI_TERMINATE_CONTEXT(APPINSTALLER_CLI_ERROR_SOURCE_NAME_DOES_NOT_EXIST);
             }
 
-            context << Workflow::OpenNamedSource(requiredSource.Details.Name);
+            context <<
```

Comment:
What I meant was:
```
context << Workflow::OpenNamedSourceForSources(requiredSource.Details.Name);
```
That way the source is never on the Sources data element and won't accidentally be used for that.